coalib/processes/Processing.py

```diff
@@ -274,18 +274,18 @@ def get_file_dict(filename_list, log_printer, allow_raw_files=False):
             if allow_raw_files:
                 file_dict[filename] = None
                 continue
-            log_printer.warn("Failed to read file '{}'. It seems to contain "
-                             'non-unicode characters. Leaving it '
-                             'out.'.format(filename))
+            logging.warn("Failed to read file '{}'. It seems to contain "
+                         'non-unicode characters. Leaving it out.'
+                         .format(filename))
         except OSError as exception:
-            log_printer.log_exception("Failed to read file '{}' because of "
-                                      'an unknown error. Leaving it '
-                                      'out.'.format(filename),
-                                      exception,
-                                      log_level=LOG_LEVEL.WARNING)
-
-    log_printer.debug('Files that will be checked:\n' +
-                      '\n'.join(file_dict.keys()))
+            log_exception("Failed to read file '{}' because of an unknown "
+                          'error. Leaving it out.'
+                          .format(filename),
```

Comment:
would indent like this:
```
log_exception("Failed to read file ..."
                  'error. Leaving it out.'.format(filename),
              exception,
              log_level=LOG_LEVEL.WARNING)
```

Reply:
`'error. ...` is not getting indented. On running coala, it aligns it back with the line above.

Reply:
kk 👍